packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -626,6 +627,173 @@
         }
       },
       "required": ["type", "image"]
+    },
+    "guidancePanel": {
+      "type": "object",
+      "description": "The guidance panel specification for the page.",
+      "additionalProperties": false,
+      "required": ["title"],
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "Guidance panel title",
+          "doNotSuggest": false
+        },
+        "backgroundImage": {
+          "type": "object",
+          "description": "Image to display in the background of this guidance panel. It should be a horizontal image and it will be fixed at the bottom of the guidance panel.",
+          "additionalProperties": false,
+          "doNotSuggest": false,
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "The image name."
+            },
+            "namespace": {
+              "type": ["null", "string"],
+              "description": "The image namespace.",
+              "defaultSnippets": [{ "label": "\"\"", "body": "${0}" }]
+            }
+          },
+          "defaultSnippets": [
+            {
+              "label": "New background image",
+              "body": {
+                "name": "${1:name}"
+              }
+            }
+          ]
+        },
+        "items": {
+          "type": "array",
+          "description": "The set of the items to display in this guidance panel.",
+          "minItems": 0,
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["type"],
+            "properties": {
+              "type": {
+                "type": "string",
+                "description": "The item type.",
+                "enum": ["Text", "Image", "LinkBox"],
+                "enumDescriptions": ["Displays text.", "Displays an image.", "Displays a link item."]
+              },
+              "text": {
+                "doNotSuggest": true
+              },
+              "variant": {
+                "doNotSuggest": true
+              },
+              "image": {
+                "doNotSuggest": true
+              },
+              "title": {
+                "doNotSuggest": true
+              },
+              "icon": {
+                "doNotSuggest": true
+              },
+              "url": {
+                "doNotSuggest": true
+              }
+            },
+            "anyOf": [
+              {
+                "type": "object",
+                "required": ["type", "text"],
+                "doNotSuggest": false,
+                "additionalProperties": false,
+                "properties": {
+                  "type": { "const": "Text" },
+                  "text": {
+                    "type": "string",
+                    "description": "The text to display in this item.",
```

Comment:
This, plus all the `LinkBox` fields (except for `url`), support `{{...}}` expressions now. In my last PR #159, I added '`This can contain {{...}} expressions.`' to the description for those.

Reply:
Ah that's true. I updated the descriptions :))